lib/broadside/configuration/deploy_config.rb

```diff
@@ -24,6 +24,7 @@ class DeployConfig < ConfigStruct
         :instance,
         :lines,
         :predeploy_commands,
+        :bootstrap_commands,
```

Comment:
i'm also unclear on what the distinction is between `bootstrap_commands` and `predeploy_commands`, given that `predeploy_commands` run before deploys...   is there some case where you want to run AWS commands before there is even a container up and running or something?